src/sentry/static/sentry/app/components/contextPickerModal.tsx

```diff
@@ -228,15 +244,24 @@ class ContextPickerModal extends React.Component<Props> {
       .filter(({status}) => status.id !== 'pending_deletion')
       .map(({slug}) => ({label: slug, value: slug}));
 
+    const customStyles = {
+      option: provided => ({
+        ...provided,
+        borderTop: `1px solid ${theme.gray1}`,
+      }),
+      menuList: provided => ({
+        ...provided,
+        paddingTop: '0px',
```

Comment:
```suggestion
        paddingTop: '0',
```
0 doesn't need units :golf:

Reply:
@markstory  ooops...merged before making this change. I'll remember this next time.